security/advancedtls/testdata/0b35a562.r1

```diff
@@ -0,0 +1 @@
+1.crl
```

Comment:
Can I know why not just read the file `1.crl` directly, instead using this file to hold the CRL file name? Is this format regulated by `X509_LOOKUP_hash_dir`?

Reply:
Correct, the hash values vary by issuer, so hard coding would create brittle tests